lib/errors.ts

```diff
@@ -122,6 +122,22 @@ const messages: {
 
 	EACCES: (e) => messages.EPERM(e),
 
+	BalenaSettingsPermissionError: () => stripIndent`
+		You don't have sufficient privileges to access balena settings.
+
+		Balena stores the session token in ${
+			os.platform() === 'win32' ? 'C:\\Users\\<user>\\_balena' : '$HOME/.balena'
+		}.
+		This error usually indicates that the user doesn't have permissions over that directory,
+		which can happen if balena was executed as the root user.
```

Comment:
_"if the CLI was executed..."_ (or _"if the balena CLI was executed...")_  instead of _"if balena was executed..."_
This is because balena is the name of the company, not the name of the CLI tool. The tool does not yet have its own proper name. It reads "The balena Command Line Interface", which is comparable to saying "The Microsoft Operating System" or "The Microsoft OS" instead of saying "Microsoft Windows".

Reply:
> because balena is [...] not the name of the CLI tool

Well, I can see how one might reasonably disagree, given that the command line reads exactly `balena`. 😁 
Still, at balena (the company) we don't refer to the balena CLI as "balena". 😀 
Btw, on balenaOS devices (host OS), `balena` is a symlink to `balena-engine`, whose name is `balenaEngine`...